NDTensors/src/lib/Sectors/test/test_category_product.jl

```diff
@@ -68,7 +78,7 @@ using Test: @test, @testset, @test_throws
     q22 = (N=U1(2),) × (J=SU{2}(5),)
 
     @test q1h ⊗ q1h == q20 ⊕ q21
-    @test q10 ⊗ q1h == [q2h]
+    @test q10 ⊗ q1h == gradedrange([q2h => 1])
```

Comment:
I think it would be good to discuss what this case should output.

Reply:
I think this choice makes sense given that it is the fusion of `CategoryProduct`s, as discussed above.